From ba915b13b92f3a625633ede43789c1ba780371af Mon Sep 17 00:00:00 2001
From: Xiangyu Chen <xiangyu.chen@windriver.com>
Date: Wed, 18 Jan 2023 03:19:07 +0000
Subject: [PATCH] prevent gendispatch.pl generating build path in code comments

gendispatch.pl will add build dir in code comments which was generated
by itself. those build dir information would cause yocto qa report
warnings like:

WARNING: libvirt-8.1.0-r0 do_package_qa: 
QA Issue: File /usr/src/debug/libvirt/8.1.0-r0/src/access/viraccessapicheckqemu.h in package libvirt-src contains reference to TMPDIR
File /usr/src/debug/libvirt/8.1.0-r0/src/access/viraccessapichecklxc.h in package libvirt-src contains reference to TMPDIR
File /usr/src/debug/libvirt/8.1.0-r0/src/access/viraccessapicheckqemu.c in package libvirt-src contains reference to TMPDIR
File /usr/src/debug/libvirt/8.1.0-r0/src/access/viraccessapichecklxc.c in package libvirt-src contains reference to TMPDIR
File /usr/src/debug/libvirt/8.1.0-r0/src/access/viraccessapicheck.h in package libvirt-src contains reference to TMPDIR
File /usr/src/debug/libvirt/8.1.0-r0/src/access/viraccessapicheck.c in package libvirt-src contains reference to TMPDIR [buildpaths]

Upstream-Status: Inappropriate [oe specific]

Signed-off-by: Xiangyu Chen <xiangyu.chen@windriver.com>
---
 src/rpc/gendispatch.pl | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/src/rpc/gendispatch.pl b/src/rpc/gendispatch.pl
index 9f5bf0e..c73a9dd 100755
--- a/src/rpc/gendispatch.pl
+++ b/src/rpc/gendispatch.pl
@@ -449,19 +449,19 @@ sub hyper_to_long
 
 if ($mode eq "aclsym") {
     print <<__EOF__;
-# Automatically generated from $protocol by gendispatch.pl.
+# Automatically generated by gendispatch.pl.
 # Do not edit this file.  Any changes you make will be lost.
 __EOF__
 } elsif ($mode eq "aclapi") {
     print <<__EOF__;
 <!--
-  -  Automatically generated from $protocol by gendispatch.pl.
+  -  Automatically generated by gendispatch.pl.
   -  Do not edit this file.  Any changes you make will be lost.
   -->
 __EOF__
 } else {
     print <<__EOF__;
-/* Automatically generated from $protocol by gendispatch.pl.
+/* Automatically generated by gendispatch.pl.
  * Do not edit this file.  Any changes you make will be lost.
  */
 __EOF__
-- 
2.33.1

